src/components/calcite-tile-select/calcite-tile-select.scss

```diff
@@ -1,18 +1,17 @@
-$spacing: $baseline/2;
-
 .root {
-  background-color: var(--calcite-ui-foreground-1);
+  @apply bg-foreground-1
+    box-border
+    cursor-pointer
+    inline-block
+    h-full
+    p-3
+    relative
+    duration-150
+    ease-in-out
```

Comment:
@julio8a the [default transition from calcite-base](https://github.com/Esri/calcite-base/blob/master/dist/_variables.scss#L2) exists [in our Tailwind config](https://github.com/Esri/calcite-components/blob/master/tailwind.config.js#L231), let's make use of it. It's applicable whenever you see `transition: $transition` in the scss file. (You can use the utility like this: `@apply transition-default`.)

Reply:
Could this be the reason test is failing?

Reply:
No, I don't think that previous fail was related